docs/UserGuide.md

```diff
@@ -103,18 +119,73 @@ Beyond individual products, you can also:
 1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds.
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some example commands you can try:
+Ta-da! *MyInsuRec* is now installed on your system.
+
+--------------------------------------------------------------------------------------------------------------------
+
+## 4. Quick start
+
+### 4.1 Basic features
+
+Are you new to *MyInsuRec*? This section is the place to start! 
+
+Here, we will be guiding you on the basic features of *MyInsuRec* and help you get familiarized with it.
+
+1. When we first launch *MyInsuRec*, *MyInsuRec* will have already been preloaded with some sample data. We will be using this sample data to get familiar with *MyInsuRec*!
+
+2. Type in `addClient n/Tommy Tan p/81234567` to add a client named 'Tommy Tan' and has a contact number '81234567'. We will see the list of clients update to include this newly added client, with his phone number! The image below shows *MyInsuRec* after adding Tommy.
+
+![AddClientTommy](images/quick-start/AddClientTommy.png)
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:**
+Both client name and their phone number are needed to add the client into *MyInsuRec*.
+</div>
+
+3. Tommy has also given us his email address, his address as well as his birthday, and we would like to add that into *MyInsuRec*. To do that, we first need to get Tommy's index from the list of clients, which is 7. Type in `editClient i/7 e/tommytan21@gmail.com a/167 Canning Road bd/25081982` and hit enter. Tommy's record in the list of clients will update with the newly inputted details. The image below shows *MyInsuRec* after updating Tommy with the additional details.
+
+![EditClientTommy](images/quick-start/EditClientTommy.png)
+
+4. We can also directly add these optional fields (email, address, etc.) when we are adding the client for the first time! So, `addClient n/Tammy Lim p/90127654 e/tammylim@hotmail.com bd/09091973` also adds in the client's email and birthday directly.
+
+<div markdown="span" class="alert alert-success">:exclamation: **Tips and tricks:**
+See [addClient](#311-adding-a-client-addclient) for more variety of optional fields!
+</div>
+
+5. Now we would like to add in an upcoming meeting with our client 'Bernice Yu' (index 2 in our sample data). We can do that by doing `addMeeting i/2 d/27102022 st/1400 et/1600 dn/Review plan` (You can replace `d/27102022` with today's date in the DDMMYYYY format)! When we hit enter, *MyInsuRec* will show us the list of meetings, including the one we just added. The image below shows *MyInsuRec* after adding the meeting with Bernice.
```

Comment:
Perhaps here instead of using `dn/Review plan` as it is quite general, you can use `dn/Review Product 2` to draw the link of why product is recorded as well.

Reply:
also at the last sentence
`The image below shows *MyInsuRec* after adding the meeting with Bernice.` can be replaced with `The image below shows a meeting list in *MyInsuRec* after adding the meeting with Bernice.`. making an emphasise on the meeting list.

Reply:
hmm, the sentence before mentioned the myinsurec showing a list of meetings actually, but i can change it to be more concise

Reply:
Ah i see. Okay i think this is fine.